### PR TITLE
Edit properties and values of EventTile on ThreadView

### PR DESCRIPTION
**Pull request by luixxiul**
Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

- Use a variable
- Remove obsolete `order` properties
- Set `padding-inline-start: 0` with a comment

type: task

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: res/css/views/rooms/_EventTile.scss [hunk 1]
```diff
@@ -861,20 +861,21 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
...
 
-        .mx_MessageTimestamp {
-            font-size: $font-10px;
+        .mx_EventTile_line,
+        .mx_ReactionsRow {
+            padding-inline-start: 0; // Cancel inherited padding value for event message and reactions row
         }
 
         .mx_ReactionsRow {
-            order: 999;
```
Comment (t3chguy) [inline res/css/views/rooms/_EventTile.scss:875]: Is this no longer required?

Comment (luixxiul) [reply res/css/views/rooms/_EventTile.scss:875]: According to [EventTile.tsx](https://github.com/matrix-org/matrix-react-sdk/blob/f427f09b8bbbee2a264721b60a0bd0d7d31b9572/src/components/views/rooms/EventTile.tsx#L1300-L1331) the reactions row are placed under the `mx_EventTile_line`.

- `mx_EventTile_roomName`
- `mx_EventTile_senderDetails`
- `mx_EventTile_line` (order: 10 !important)
- `mx_ReactionsRow` (order: 999)

Also on DOM tree:

Untitled

The declaration was added with https://github.com/matrix-org/matrix-react-sdk/pull/6997/commits/4c9b0ffbba27de08d3543252c5fa90fb276bfcb9#diff-bd83194d78de871bc4ad1c4e8d43102c4368481bbb90705152d3d17fea813919R785, which fixed something for [this](https://github.com/matrix-org/matrix-react-sdk/pull/6997/commits/4c9b0ffbba27de08d3543252c5fa90fb276bfcb9#diff-9ff5535291b7c2e655ccd3d02c4f632b6f2dc9c30072514641e31c2a8751c57eR1265).

As the reactions row can be displayed without the declaration, it should be able to be safely removed along with `order: 10 !important` for `mx_EventTile_line`.

Comment (luixxiul) [reply res/css/views/rooms/_EventTile.scss:873]: Also those declarations are not set for event tiles on `TimelineCard`, and I am not quite sure why they would be exclusively requited for `ThreadView`.

**Review by t3chguy — APPROVED** [review]
Looks sane, thanks!!

**Pull request merged**